Limelight/UIAppView.m

```diff
@@ -32,7 +32,11 @@ - (id) initWithApp:(TemporaryApp*)app cache:(NSCache*)cache andCallback:(id<AppC
         noImage = [UIImage imageNamed:@"NoAppImage"];
     }
     
+#if TARGET_OS_TV
+    _appButton = [UIButton buttonWithType:UIButtonTypeSystem];
```

Comment:
Do we get some sort of styling difference using `UIButtonTypeSystem`?

Reply:
Without `UIButtonTypeSystem`, I wasn't getting the tvOS style "zoomed in, rotates a little on touch" style selectors that are used throughout the system.